core/controller/src/main/scala/whisk/core/controller/Entities.scala

```diff
@@ -65,14 +65,25 @@ protected[controller] trait ValidateRequestSize extends Directives {
   protected val fieldDescriptionForSizeError = "Request"
 }
 
+protected trait CustomHeaders extends Directives {
+
+  /** Add activation ID in headers */
+  protected def respondWithActivationIdHeader(activation: WhiskActivation): Directive0 =
```

Comment:
Do we need to override this method?
Wouldn't it be easier to always pass the activationId? You have it anyway, if you have the activation.

Reply:
Updated accordingly